docs/working-with-projects/working-with-images/deleting-artifact.md

```diff
@@ -3,7 +3,7 @@ title: Deleting Artifacts
 weight: 75
 ---
 
-Harbor v2.0 now supports OCI images and OCI image indexes (https://github.com/opencontainers/image-spec/blob/master/image-index.md). An OCI image index (or OCI index) is a higher level manifest which points to a list of image manifests, ideal for one or more platforms.  Both the index itself and the images referenced within are referred to as artifacts in Harbor parlance. An OCI index could hold another OCI index and so on and so forth.  For any artifact referenced by an OCI index, the referenced artifact is known as the child artifact and the OCI index referencing the artifact is known as the parent artifact.  We can also say that the child artifact belongs to the parent artifact or is a part of the parent artifact.  
+Harbor v2.0 supports OCI images and OCI image indexes (https://github.com/opencontainers/image-spec/blob/master/image-index.md). An OCI image index (or OCI index) is a higher level manifest which points to a list of image manifests, ideal for one or more platforms.  Both the index itself and the images referenced within are called artifacts in Harbor. An OCI index can hold another OCI index and so on and so forth. For any artifact referenced by an OCI index, the referenced artifact is known as the child artifact and the OCI index referencing the artifact is known as the parent artifact. The child artifact belongs to the parent artifact or is a part of the parent artifact.  
```

Comment:
so I don't think its unequivocally true or accepted that child 'belongs' to parent, because for ex. when you delete the parent artifact, the child will not get deleted.  which is why I added 'we can say', maybe we can use 'it's common to say' ?

Reply:
So, you are saying that 'we can say' or that 'other people say' that the child artifact belongs to the parent, even though it is inaccurate to say such a thing. Perhaps it is better to completely remove this statement. Even if it were true, it doesn't really add any clarity to the discussion.

Reply:
apologies for my tardiness.  I meant it's conventionally acceptable to say this but we are effectively caveating it with 'we can say'.  Whereas the edit comes off a bit too strong because 'parent' and 'child' are not widely adopted terms in OCI, not in object oriented programming for ex. 

it's not a big deal, but I prefer the original

Reply:
The preceding content defines the parent/child relationship ... which appears to be more of a 'linked' relationship than a true parent/child relationship. The terms 'parent' and 'child' are adequately established for this content - explaining what they actually mean, rather than how one might misconstrue them. 
So the last sentence is really unnecessary. 
@xaleeks Don't you think it adds potential confusion for future expansion if we imply ownership when it is not true?

Reply:
I deleted the last sentence for now. 
If you accept the reasoning, you can simply resolve the conversation.
If you really want it added back in, I'll put it back and drop the argument.

Reply:
lgtm